apis/metal3.io/v1alpha1/baremetalhost_types.go

```diff
@@ -219,6 +219,62 @@ type BMCDetails struct {
 	DisableCertificateVerification bool `json:"disableCertificateVerification,omitempty"`
 }
 
+// HardwareRAIDVolume defines the desired configuration of volume in hardware RAID
+type HardwareRAIDVolume struct {
+	// Size (Integer) of the logical disk to be created in GiB.
+	// If unspecified or set be 0, the maximum capacity of disk will be used for logical disk.
+	// +kubebuilder:validation:Minimum=0
+	SizeGibibytes *int `json:"sizeGibibytes,omitempty"`
+
+	// RAID level for the logical disk. The following levels are supported: 0;1;1+0.
+	// +kubebuilder:validation:Enum="0";"1";"1+0"
```

Comment:
Same question as below. The proposed Redfish RAID supports 5 and 6 (and even 5+0 and 6+0 IIRC)

Reply:
I think i can add other RAID level to here.

Reply:
Yep (I'm fine with a follow-up)